src/spectrecoff-cli/commands/Output.fs

```diff
@@ -98,6 +98,17 @@ type OutputExample() =
             C $"""or use the constants provided by Spectre {Emoji.Known.Ghost} inline."""
             NewLine
         ] |> toConsole
+    
+        // Use extensions on the payload
+        let payload = Many [ NL; P "Pinted"; E "Using"; NL; C "... the Extension!"]
+        payload.toConsole
```

Comment:
I wonder, should that approach maybe be extended to the other types, like `ChartItem.toBarChart`? You are right with the discoverability, also it looks nicer :)

Reply:
Your example does not really make sense to me, since an item is not converted to a chart. So - where it makes sense - we can do it, but I'd keep it minimal.